backend/src/tasks/LookingGlass.ts

```diff
@@ -0,0 +1,286 @@
+import {
+  Organization,
+  Domain,
+  Vulnerability,
+  connectToDatabase
+} from '../models';
+import { CommandOptions } from './ecs-client';
+import got from 'got';
+import { plainToClass } from 'class-transformer';
+import saveVulnerabilitiesToDb from './helpers/saveVulnerabilitiesToDb';
+import { exit } from 'process';
+import saveDomainsToDb from './helpers/saveDomainsToDb';
+
+interface LGCollectionsResponse {
+  children: any[];
+  createdAt: string;
+  createdBy: string;
+  id: string;
+  name: string;
+  ticScore: number;
+  updatedAt: string;
+  updatedBy: string;
+}
+
+interface LGThreatResponse {
+  totalHits: number;
+  results: {
+    firstSeen: number;
+    lastSeen: number;
+    sources: string[];
+    right: {
+      ticClassificationScores: number[];
+      threatId: string;
+      classifications: string[];
+      ticScore: number;
+      ticCriticality: number;
+      ticSourceScore: number;
+      ticObsInfluence: number;
+      ref: {
+        type: string;
+        id: string;
+      };
+      threatCategories: string[];
+      name: string;
+      type: string;
+      threatName: string;
+    };
+    left: {
+      collectionIds: string[];
+      classifications: string[];
+      ticScore: number;
+      owners: string[];
+      ref: {
+        type: string;
+        id: number;
+      };
+      name: string;
+      type: string;
+      threatNames: string[];
+      locations: {
+        city: string;
+        country: string;
+        region: string;
+        geoPoint: {
+          long: number;
+          lat: number;
+        };
+        countryName: string;
+        sources: string[];
+        country2Digit: string;
+        lastSeen: number;
+      }[];
+      ipv4: number;
+      asns: number[];
+      threatIds: string[];
+      cidrv4s: string[];
+    };
+    ref: {
+      type: string;
+      right: {
+        type: string;
+        id: string;
+      };
+      left: {
+        type: string;
+        id: string;
+      };
+    };
+  }[];
+}
+
+async function getOrg(organizationId: string | undefined) {
+  await connectToDatabase();
+  if (typeof organizationId === 'string') {
+    const organization = await Organization.findOne(organizationId);
+    if (organization) {
+      return organization;
+    } else {
+      console.log('No organization found');
+      exit;
+    }
+  } else {
+    console.log('No OrganizationId provided');
+    exit;
+  }
+}
+
+async function collectionByWorkspace() {
```

Comment:
Let's rename this to `getCollectionForCurrentWorkspace` -- so it's clear what it does